specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementApproveOrRejectPrivateEndpointConnection.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "privateEndpointConnectionName": "privateEndpointConnectionName",
+    "privateEndpointConnectionRequest": {
+      "Id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/privateEndpointConnections/connectionName",
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description": "The Private Endpoint Connection is approved."
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/tempgroup?api-version=2021-04-01-preview&asyncId=5c730e343244df1b9cb56e85&asyncCode=201"
+      }
+    },
+    "200": {
+      "body": {
+        "name": "privateEndpointConnectionName",
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/privateEndpointConnections/connectionName",
+        "type": "Microsoft.ApiManagement/service/privateEndpointConnections",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "privateEndpoint": {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/privateEndpoints/privateEndpointName"
```

Comment:
>/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/privateEndpoints/privateEndpointName [](http://example.com/codeflow?start=19&length=104)

I don't see it in the request parameters. Where does it come from?

Reply:
hi it's defined in this file: 
![image](https://user-images.githubusercontent.com/29927264/127920419-00dc4b86-aa36-4760-b56f-2e9c654ec669.png)

Reply:
I guess that id is supposed to be a pre-existing privateEndpoint. Is that right? If so, maybe you can add creating it into the example.

Reply:
creating this resource/id is in network's examples, other teams onboarded to the feature didn't provide this example (since it's part of network team's example already)